rune/libcontainer/configs/enclave.go

```diff
@@ -7,7 +7,10 @@ const (
 )
 
 type Enclave struct {
-	Type string `json:"type"`
-	Path string `json:"path"`
-	Args string `json:"args,omitempty"`
+	Type                    string `json:"type"`
+	Path                    string `json:"path"`
+	Args                    string `json:"args,omitempty"`
+	RaClientSpid            string `json:"ra_client_spid"`
+	RaClientSubscriptionKey string `json:"ra_client_subscription_key"`
+	RaClientQuoteType       string `json:"ra_client_quote_type"`
```

Comment:
`client` is useless.

In addition, add `omitempty` for each new RA attributes, cause they are optional.